### DISCUSSION TITLE
personal-summary/cs/18-网络工程-PYQ-保研-@IIE(CAS)

### DISCUSSION THREAD

**Discussion by giscus** [Hainanu-Application/Hainanu-Application.github.io#5] [Announcements]
# /personal-summary/cs/18-网络工程-PYQ-保研-@IIE(CAS)

A magical documentation generator.

https://hainanu-application.github.io/#/personal-summary/cs/18-%E7%BD%91%E7%BB%9C%E5%B7%A5%E7%A8%8B-PYQ-%E4%BF%9D%E7%A0%94-@IIE(CAS)

**Comment by peng-yq** [comment]
PYQ牛逼！！！！！

**Comment by Haodong-Lei-Ray** [comment]
学长英语怎么样？（哇库哇库）

**Reply by peng-yq** [reply]
一般般

**Reply by Haodong-Lei-Ray** [reply]
冒昧地问问cet6多少分？

**Reply by peng-yq** [reply]
450
